packages/server/tests/acceptance/cacheService.spec.ts

```diff
@@ -103,4 +103,59 @@ describe('@cache-service Acceptance Tests for shared cache', function () {
     const cachedData = await otherServiceInstance.getAsync(dataLabel, CALLING_METHOD);
     expect(cachedData).to.deep.eq(DATA, 'cached data is read correctly by other service instance');
   });
+
+  describe('fallback to local cache in case of Redis error', async () => {
+    const dataLabel = `${DATA_LABEL_PREFIX}_redis_error`;
+
+    let currentRedisEnabledEnv;
+    let cacheService;
+
+    beforeEach(async () => {
```

Comment:
Let's not slow down the tests unnecessarily:

```suggestion
    before(async () => {
```